crates/loans/src/lend_token.rs

```diff
@@ -141,7 +141,10 @@ impl<T: Config> Pallet<T> {
         let reducible_underlying_amount = reducible_supply_amount.convert_to(underlying_id)?.amount();
 
         let exchange_rate = Self::exchange_rate(underlying_id);
-        let amount = Self::calc_collateral_amount(reducible_underlying_amount, exchange_rate)?;
+        let amount = FixedU128::from_inner(reducible_underlying_amount)
```

Comment:
How come you don't need to convert `reducible_underlying_amount` using `checked_from_int`?

Reply:
Because it's `from_inner` rather than `from`. So a value of `1` results in a value of 1e-18